middlewares/osio/osio_che_test.go

```diff
@@ -217,3 +220,7 @@ func startServer(url string, handler func(w http.ResponseWriter, r *http.Request
 	}
 	return
 }
+
+func testSrvAccToken(token string) (bool, error) {
+	return true, nil
+}
```

Comment:
Test impl for SrvAccTokenChecker, always return true. This will avoid executing actual SrvAccTokenChecker() which required jwt_token_parsing works during test execution.